spec/helpers/search_helper_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SearchHelper, type: :helper do
+  describe '#classes_for_sign_attribute' do
+    context 'when provided with a true value for main' do
+      it 'returns the main_image class as part of the response string' do
+        expect(helper.send(:classes_for_sign_attribute, :location, true)).to eq('image rounded main_image')
+      end
+    end
+
+    context 'when provided with an :handshape attribute' do
+      it 'returns the transition class as part of the response string' do
+        expect(helper.send(:classes_for_sign_attribute, :handshape, false)).to eq('image rounded transition')
+      end
+    end
+
+    context 'when provided with a false value for main and a attribute not matching :handshape' do
+      it 'returns the default class string' do
+        expect(helper.send(:classes_for_sign_attribute, nil, false)).to eq('image rounded ')
```

Comment:
how about splitting the classes by ' ' and then checking that `image` and `rounded` were included in there?
Then this test is less likely to break in future css/classname changes.

Reply:
there might be a smart railsy-way to check for classes on an element.